api/src/database/db.test.ts

```diff
@@ -1,12 +1,20 @@
 import { expect } from 'chai';
 import { describe } from 'mocha';
 import * as pg from 'pg';
-import Sinon from 'sinon';
+import Sinon, { SinonStub } from 'sinon';
+import SQL from 'sql-template-strings';
 import { SYSTEM_IDENTITY_SOURCE } from '../constants/database';
 import { HTTPError } from '../errors/custom-error';
 import { setSystemUserContextSQL } from '../queries/database/user-context-queries';
 import * as db from './db';
```

Comment:
Can we delete this line?

Reply:
Its still used in a stub further down

Reply:
In order to tub out its content we have to import it this way.